Student/Student/StudentAppDelegate.swift

```diff
@@ -299,6 +299,11 @@ extension StudentAppDelegate: Core.AnalyticsHandler {
     }
 
     func handleEvent(_ name: String, parameters: [String: Any]?) {
+        Heap.track(name, withProperties: parameters)
+        PageViewEventController.instance.logPageView(
+            name,
+            attributes: parameters as? [String: String] ?? [:] // if there's a type mismatch attributes will be lost
```

Comment:
Would it make sense to create another `logPageView` method which expects `[String: Any]? = [:]` and do the casting there?

Reply:
Move the logic there.